product/models.py

```diff
@@ -0,0 +1,125 @@
+from django.db import models
+
+class Product(models.Model):
+    name           = models.CharField(max_length = 100)
+    price          = models.DecimalField(max_digits = 10, decimal_places = 2)
+    description    = models.CharField(max_length = 100)
+    main_image_url = models.URLField()
+    category       = models.ManyToManyField('category', through = 'productcategory')
+    item           = models.ManyToManyField('item', through = 'productitem')
+    type           = models.ManyToManyField('type', through = 'producttype')
+    Technology     = models.ManyToManyField('technology', through = 'producttechnology')
```

Comment:
class의 경우만 시작을 대문자로 해요 가빈님~~
```suggestion
    technology     = models.ManyToManyField('technology', through = 'producttechnology')
```